packages/core/src/utils.ts

```diff
@@ -34,4 +34,12 @@ export function mergeDeep<T>(target: T, source: Partial<T>): Partial<T> {
     return target
 }
 
-export const camelify = t => t.replace(/^-+/, '').replace(/-./g, ([, l]) => l.toUpperCase())
+/**
+ * Will turn any string kebab-case into camel-case.
+ *
+ * @example
+ * camelify('dark-blue') // darkBlue
+ * camelify('--color-light-pink') // colorLightPink
+ * camelify('yellow') // yellow
+ */
+export const camelify = (t: string): string => t.replace(/^-+/, '').replace(/-./g, ([, l]) => l.toUpperCase())
```

Comment:
`toCamelCase` could be a better name IMO (so we can follow `toUpper/LowerCase` naming), but... leave it. I don't want (and I shouldn't) touch this method.

Reply:
You can, that's fine.

Reply:
changed